### PR TITLE
Update pushlists

### PR DESCRIPTION
**Pull request by m4rc1e**
cc @chrissimpkins

### CODE REVIEW

File: to_production.txt [hunk 1]
```diff
@@ -1,16 +1,19 @@
 # New
-ofl/suse # https://github.com/google/fonts/pull/8044
+ofl/eduauvicwantdots # https://github.com/google/fonts/pull/8020
```
Comment (chrissimpkins) [inline to_production.txt:2]: Do we need the tagging CSV push for any of these new families?

**Review by chrissimpkins — CHANGES_REQUESTED** [review]

**Comment by m4rc1e** [comment]
SUSE looks good. I don't think the dots have em so I can add them on Monday.
```
(venv) Marcs-Air-2:fonts marcfoley$ cat tags/all/families.csv | grep "SUSE"
SUSE,/Expressive/Calm,80
SUSE,/Expressive/Futuristic,70
SUSE,/Expressive/Happy,70
SUSE,/Expressive/Innovative,80
SUSE,/Sans/Geometric,100
SUSE,/Sans/Humanist,40
SUSE,/Sans/Superellipse,50
```

**Comment by chrissimpkins** [comment]
Thanks Marc! Let me confirm that Garret can integrate the Workspace denylist addition on Sixtyfour before we merge this.  Should happen early next week.  I will speak with him on Monday.

**Comment by m4rc1e** [comment]
Thanks Chris. The Edu family has been added in https://github.com/google/fonts/pull/8153

File: to_production.txt (hunk 1 above)
Comment (emmamarichal) [reply to_production.txt:2]: @m4rc1e?

**Review by emmamarichal — APPROVED** [review]

**Pull request merged**